akvo/rsr/spa/app/modules/results/period.jsx

```diff
@@ -213,72 +216,82 @@ const Period = ({ setResults, period, measure, treeFilter, statusFilter, increas
         </div>
       }
     >
-      {indicator.type === 1 &&
-      <div className="graph">
-        <div className="sticky">
-          {disaggregations.length > 0 && <DsgOverview {...{ disaggregations, targets: period.disaggregationTargets, period, editPeriod, values: updates.map(it => ({ value: it.value, status: it.status })), updatesListRef, setHover }} />}
-          {disaggregations.length === 0 && <Timeline {...{ updates, indicator, period, pinned, updatesListRef, setHover, editPeriod }} />}
-          {baseline.value &&
-            <div className="baseline-values">
-              <div className="baseline-value value">
-                <div className="label">{t('baseline value')}</div>
-                <div className="value">{baseline.value}{indicator.measure === '2' && <small>%</small>}</div>
-              </div>
-              <div className="baseline-value year">
-                <div className="label">{t('baseline year')}</div>
-                <div className="value">{baseline.year}</div>
-              </div>
-            </div>
-          }
-        </div>
-      </div>
-      }
-      <div className={classNames('updates', { qualitative: indicator.type === 2 })} ref={(ref) => { updatesListRef.current = ref }}>
-        <Collapse accordion activeKey={pinned} defaultActiveKey="0" onChange={handleAccordionChange} className="updates-list">
-          {sortedUpdates.map((update, index) =>
-            <Panel
-              key={index}
-              className={classNames({ 'new-update': update.isNew, hidden: editing !== -1 && editing !== index, 'pending-update': update.status === 'P' })}
-              header={
-                <Aux>
-                  <div className="value-container">
-                    {indicator.type === 1 && editing !== index && <div className={classNames('value', { hovered: hover === updates.length - 1 - index || Number(pinned) === index })}>{String(update.value).replace(/\B(?=(\d{3})+(?!\d))/g, ',')}{indicator.measure === '2' && <small>%</small>}</div>}
+      {pinned === '0' && indicator?.description?.length > 0 && (
```

Comment:
Actually, I only made changes in this section to showing an indicator description.

Reply:
For next time, please split the commit into multiple commits to make it more clearer and easier to review.